### ISSUE TITLE
Device connects, but doesn't output audio

### ISSUE THREAD

**Issue by FluroNX** [plutooo/nx-btred#2]
Device is TAOTRONICS SoundLiberty 79
on 12.0.1

**Comment by plutooo** [comment]
Sometimes after pairing you may need to turn off and on the earbuds once, does that help? It should only be necessary on first pairing

**Comment by nicholemattera** [comment]
I was having the same issue with my AirPod Pros. My Switch is on 12.0.0 using Atmosphere 0.19.1. I turned them off and back on again and didn't get any audio is qLaunch (or from games), but when I ran homebrew I was deafened by highly distorted audio.

**Comment by FluroNX** [comment]
> Sometimes after pairing you may need to turn off and on the earbuds once, does that help? It should only be necessary on first pairing

yeah no. didn't work

**Comment by plutooo** [comment]
Added a clause about limitations to the README, does it help?

**Comment by cucholix** [comment]
Same issue with my Sennheiser HD350, both devices pair correctly, but there’s no sound coming from the headphones neither Switch, had to reboot ams to restore audio from Switch speakers

**Comment by JonJaded** [comment]
Just wanted to let you know, I think this only works on games that feature screen recording (last 30 seconds) from what I've tested. I'll verify in a sec.

**Comment by plutooo** [comment]
@cucholix Did you try entering sleep-mode and then starting up the console again?

**Comment by cucholix** [comment]
> @cucholix Did you try entering sleep-mode and then starting up the console again?

I did, there’s no audio during menu, but once I started a game a high pitched sound almost left me deaf lol, had to turn the volume all the way down, only static sound comes from the headphones.

Upon other test, putting the Switch to sleep then waking up, then entering a game the audio is normal the first 5 seconds, then the static sound kicks in.

I disable Bluetooth from devices that could connect to my headphones.
